openssl/Cargo.toml

```diff
@@ -17,6 +17,7 @@ all-features = true
 v101 = []
 v102 = []
 v110 = []
+v111 = ["v110"]
```

Comment:
Why does this inherit?

Reply:
Because 1.1.1 is a superset of 1.1.0, as far as I'm aware. An alternative would be to have v11x and make v111 and v110 both imply it, but I think v110 and v11x would end up functionally equivalent, which feels a bit silly.